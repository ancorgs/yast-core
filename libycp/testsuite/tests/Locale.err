Parsed:
----------------------------------------------------------------------
_("Everybody likes Linux!")
----------------------------------------------------------------------
[Interpreter] :0 virtual evaluateLocale() !
Parsed:
----------------------------------------------------------------------
_("%d m$-windows system deleted", "%d m$-windows systems deleted", 42)
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
_("%d linux system installed", "%d linux systems installed", (40 + 2))
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
{
  locale `a = "abc";
  if (is (a, locale)) {
  return a;
}
  return false;
}
----------------------------------------------------------------------
[Scanner] tests/Locale.ycp:32 Syntax error: empty locale string
