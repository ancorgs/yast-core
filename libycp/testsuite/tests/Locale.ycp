
# ---------------------------------------------------------
#
#  Filename:    Locale.ycp
#
#  Purpose:     test cases for YCP test
#
#  Creator:     jd@suse.de
#
#  Maintainer:  jd@suse.de
#
# ---------------------------------------------------------


# --------- YCP_Locale

( _("Everybody likes Linux!") )

( _("%d m$-windows system deleted", "%d m$-windows systems deleted", 42) )

( _("%d linux system installed", "%d linux systems installed", 40+2) )

{ // string -> locale propagation

  locale a = "abc";
  if (is (a, locale))
	return a;
  return false;
}

# error
(_(""))
