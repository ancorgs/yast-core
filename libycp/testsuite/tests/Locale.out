(_("Everybody likes Linux!"))
(_("%d m$-windows system deleted", "%d m$-windows systems deleted", 42))
(_("%d linux system installed", "%d linux systems installed", 42))
(_("abc"))
